src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -156,12 +156,6 @@ public Payment getPlatformPayment(String name) throws PreActionException {
 
             //if not rethrow the exception and update the db
             if (throwable != null) {
-                try {
```

Comment:
We cannot. Because we have no information what happend. The Platformcode has to recover itself.